pandas/_libs/index_class_helper.pxi.in

```diff
@@ -30,6 +30,8 @@ dtypes = [('Float64', 'float64', 'float64_t'),
 
 cdef class {{name}}Engine(IndexEngine):
 
+    _dtype = '{{dtype}}'
```

Comment:
For testing purposes (see test_engine.py), we need the dtype that is compatible with this engine + it's useful information to have in general.